python/usb_console/radio_session.py

```diff
@@ -10,6 +10,7 @@
 import base64
 import os
 import email
+import requests
```

Comment:
Remember to add to requirements.txt as part of the venv setup (check out the other comment I've made about this on gsw/main.py)